pages/students.tsx

```diff
@@ -138,9 +128,9 @@ We emulate the Agile Workflow to give our developers some insight on industry st
 
 We host both internal and public events. To find our public events, follow us on [Facebook](https://www.facebook.com/TritonSE) and [Instagram](https://www.instagram.com/ucsd_tse/). You can also find a list of our past and upcoming events [here](/events).
 
-### How is TSE operating during the ongoing pandemic?
+### How did TSE operate during the pandemic?
 
-We are working in a hybrid model. Our last recruitment season was completely online and we plan to continue the online structure this year. We hope to have a hybrid recruitment process next year. We will do our best to provide as much support as possible through in person Information Sessions and various panels, but all interviews and phone screens will continue to be remote.
+We worked in a hybrid model during the pandemic. Now, we are continuing our hybrid model while increasing in person activities. Our last recruitment season was completely online and we plan to continue the online structure this year. We will do our best to provide as much support as possible through in person Information Sessions and various panels, but all interviews and phone screens will continue to be remote.
 
 All TSE team meetings and work are conducted in a hybrid structure at the moment. We will be continuing regular virtual & in person workshops and social events. This year, all of our events are primarily in person. Please visit our website and Facebook page for the most up-to-date information.
```

Comment:
idk the code style for adding a link, but @justinyaodu can you link to the events page here?

Please visit our website and Facebook page for the most up-to-date information. >>> For the most up-to-date information, visit our Facebook page or check out upcoming events [here](https://tse.ucsd.edu/events/).